src/components/RenderRouter/ContentItem.scss

```diff
@@ -176,15 +176,19 @@
 
 @media (min-width: 320px) and (max-width: 767px) {
```

Comment:
I was just thinking we can probably get rid of the min-width, so it works on iPhone SE and the older phones...

Reply:
Sounds good. It'd be a good "catch all" solution for sure. I'll remove the min-width then from the style sheets later tonight.